base/shell/explorer/util.cpp

```diff
@@ -228,6 +229,26 @@ GetVersionInfoString(IN LPCWSTR szFileName,
                         }
                     }
                 }
+                if(bRet == FALSE && cbTranslate >= sizeof(LANGCODEPAGE)) {
```

Comment:
Please review all the formatting of your new code. (No tabs, 4-space indent, see also https://reactos.org/wiki/Coding_Style, etc. )

Reply:
This should be fixed i think